resources/dicts/thoughts/dead/darkforest/exiled.json

```
@@ -1,3 +1,63 @@
 [
-    
-]
\ No newline at end of file
+  {
+    "id": "exiled_df_dead_thoughts",
+    "thoughts": [
+      "Is thinking of the dissaproving glares of {PRONOUN/m_c/poss} old Clanmates when they exiled {PRONOUN/m_c/object} while {PRONOUN/m_c/subject} train",
```

```suggestion
      "Is thinking of the disapproving glares of {PRONOUN/m_c/poss} ex-Clanmates when they exiled {PRONOUN/m_c/object} while {PRONOUN/m_c/subject} {VERB/m_c/train/trains}",
```
sentence feels a little meandering, maybe something like `Thinks of the disapproving glares of {PRONOUN/m_c/poss} ex-Clanmates while {PRONOUN/m_c/subject} {VERB/m_c/train/trains}` might be better?

---

definitely much more concise, ty

---

added the suggested wording